lib/pushmi_pullyu/queue.rb

```diff
@@ -0,0 +1,53 @@
+require 'redis'
+
+# Quick and dirty take on sorted sets
+
+# 1) Create a sorted set in Redis (https://redis.io/topics/data-types). Call it preservation_queue
+#
+# 2) In GenericFile add an after_save that:
+#    - determines a monotonically increasing "score". Obvious scores would be either the time in seconds/milliseconds
```

Comment:
Yeah agreed. Thansk for explanation, makes sense.

Don't think we need the granularity of `.to_f` gives. `.to_i` is probably more then good enough with the reasons you gave.